### PR TITLE
Improve the build a bit

### PR DESCRIPTION
**Pull request by ctron**
This reduces the size of the release WASM from ~6MiB to ~4MiB. It also shows an initial loading animation while the WASM is loading.

### CODE REVIEW

File: spog/ui/index.html [hunk 1]
```diff
@@ -3,7 +3,31 @@
...
+        #loading {
+            background-color: white;
+            position: fixed;
+            top: 0; left: 0;
+            width: 100vw;
+            height: 100vh;
+            overflow: hidden;
+        }
+        #loading-background {
+            background-image: url("images/chicken-svgrepo-com.svg");
```
Comment (carlosthe19916) [inline spog/ui/index.html:18]:
```suggestion
            background-image: url("assets/images/chicken-svgrepo-com.svg");
```

**Review by carlosthe19916 — CHANGES_REQUESTED** [review]
Thanks @ctron I do see a reduction of the size of the wasm file from 6.5MB to 5.6MB. I just added a minor code suggestion changes because the Chicken logo is not actually rendered while loading the page

## With this PR
Screenshot from 2023-07-13 09-54-00

## Without this PR
Screenshot from 2023-07-13 09-54-09

**Comment by ctron** [comment]
> Thanks @ctron I do see a reduction of the size of the wasm file from 6.5MB to 5.6MB.

That's interesting, because for me it looks like this:

```
-rw-r--r--.  1 jreimann jreimann 4.1M Jul 13 10:14 spog-ui-7a2e21e609f5c4b6_bg.wasm
-rw-r--r--.  1 jreimann jreimann  47K Jul 13 10:14 spog-ui-7a2e21e609f5c4b6.js
```

File: spog/ui/index.html (hunk 1 above)
Comment (ctron) [reply spog/ui/index.html]: Indeed. Fixed, thx!

**Review by carlosthe19916 — APPROVED** [review]

**Comment by carlosthe19916** [comment]
> > Thanks @ctron I do see a reduction of the size of the wasm file from 6.5MB to 5.6MB.
> 
> That's interesting, because for me it looks like this:
> 
> ```
> -rw-r--r--.  1 jreimann jreimann 4.1M Jul 13 10:14 spog-ui-7a2e21e609f5c4b6_bg.wasm
> -rw-r--r--.  1 jreimann jreimann  47K Jul 13 10:14 spog-ui-7a2e21e609f5c4b6.js
> ```
I have no idea why we see different sizes. Let's see what we will see once this PR is merged. I've just approved this PR because it looks good to me.

**Pull request merged**